pkg/api/operators/apis/trial/v1alpha2/types.go

```diff
@@ -0,0 +1,104 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha2
+
+import (
+	pb "github.com/kubeflow/katib/pkg/api"
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type TrialSpec struct {
+	// Key-value pairs for Parameters and Assignment values
+	SuggestionParameters []pb.SuggestionParameter `json:"suggestionParameters"`
```

Comment:
SuggestionParameter is a parameter for Suggestion service. https://github.com/kubeflow/katib/blob/master/pkg/api/api.proto#L327

This should be `Parameter` https://github.com/kubeflow/katib/blob/master/pkg/api/api.proto#L273
And the name `SuggestionParameters` is confusing. How about only `ParameterSet` or `ParameterList`?

Reply:
I really liked Alexandra's suggestion of ParameterAssignment. I agree SuggestionParameters is confusing.